src/Ocean/CartesianDomains/CartesianDomains.jl

```diff
@@ -0,0 +1,9 @@
+module CartesianDomains
```

Comment:
I'm wondering if this belongs in `src/Common/` in case other groups may find it useful.

Reply:
Perhaps they would --- that's more up to you (and others) than me. This functionality is still a bit experimental but I wouldn't mind if others wanted to try it out and gave feedback.